rllib/algorithms/marwil/marwil.py

```diff
@@ -103,9 +103,10 @@ def __init__(self, trainer_class=None):
         # the same line.
         self.input_ = "sampler"
         # Use importance sampling estimators for reward.
-        self.off_policy_estimation_methods = [
-            ImportanceSampling, WeightedImportanceSampling
-        ]
+        self.off_policy_estimation_methods = {
+            "is": {"type": ImportanceSampling},
```

Comment:
??
can you respond to @avnishn's comments please?

Reply:
So this is the pattern I was following, from Sven's comment earlier:
```
config:
  off_policy_estimators:
    my_is:
      type: ImportanceSampling
      ctor_kwarg1: [some value1]
    dr:
      type: DoublyRobust
      ctor_kwarg1: [some value]
      ctor_kwarg2: [some other value]
```

So the name e.g. `my_is` is the string under which the estimator's results are saved. `type` is the actual class or classpath for the estimator. For IS and WIS, this isn't really relevant, since they don't take any other arguments beyong policy and gamma.

However for e.g. DR, you can do something like this:
```
off_policy_estimation_methods: {
            "dr_qreg": {
                "type": DoublyRobust,
                "q_model_type": "qreg",
            },
            "dr_fqe_tau_1": {
                "type": DoublyRobust,
                "q_model_type": "fqe",
                "tau": 0.01,
            },
            "dr_fqe_tau_5": {
                "type": DoublyRobust,
                "q_model_type": "fqe",
                "tau": 0.05,
            },
}
```
And so on. I've actually updated the `test_ope` test accordingly, although it now takes ~45 minutes to run (Should come back down to ~10 minutes when we put in parallel eval/k_fold_cv/Ray AIR)

Reply:
@gjoliver ^ Does this answer your questions?

Reply:
this makes a lot of sense. thanks.
I think we should document this though, for example, add your DR estimate configs as an example to the doc?
and here, I wonder if we can just say:
```
# Use importance sampling estimators for reward.
# off_policy_estimation_methods is a map from arbitrary string names to Off-Policy Evaluation configs.
# Metrics for each OPE config will show up under corresponding key name.
```
?
Thanks.